### ISSUE TITLE
accessibility: Add marking aria for buttons view + labels for mcq item marking

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2254]
### Subject of the issue
Buttons view central icon doesn't have aria, mcq items need marking aria

**Comment by oliverfoster** [comment]
done
